# ====> Lint (error only mode) testing
######################################################################
Using config file /home/avg/.pylintrc

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)



# ====> Unit testing
######################################################################
......
----------------------------------------------------------------------
Ran 6 tests in 0.007s

OK


# ====> Integration testing
######################################################################
Discover Test Args: None
* Test Suite args: None
* Running test suite 'AVG QA-12 integration tests'
  - Running Database interacation test : [  OK  ]
Ran 1 tests, 1 succeeded, 0 failed (0 expected), 0 skipped

Summary: Ran 1 tests, 1 succeeded, 0 failed (0 expected), 0 skipped


# ====> Smoke testing
######################################################################
curl to http://unoume-qa12.devops.rebrain.srwx.net status:
HTTP/1.1 200 OK
Server: nginx/1.14.0 (Ubuntu)
Date: Tue, 11 Aug 2020 14:54:29 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 1186
Connection: keep-alive



# ====> API testing
######################################################################
Discover Test Args: None
* Test Suite args: None
* Running test suite 'AVG QA-12 API tests'
  - Running Cleanup page API test        : [  OK  ]
  - Running Root page API test           : [  OK  ]
  - Running Calc page API test #1        : [  OK  ]
  - Running Calc page API test #2        : [  OK  ]
  - Running Results_secret page API test : [  OK  ]
  - Running Result/<id> page API test    : [  OK  ]
Ran 6 tests, 6 succeeded, 0 failed (0 expected), 0 skipped

Summary: Ran 6 tests, 6 succeeded, 0 failed (0 expected), 0 skipped


# ====> Interface testing
######################################################################
.F
======================================================================
FAIL: test_cleanup_interface (iface_tests.TestCalc)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/avg/qa-system/tests/iface_tests.py", line 45, in test_cleanup_interface
    self.assertTrue(cleanup_form)
AssertionError: None is not true

----------------------------------------------------------------------
Ran 2 tests in 0.033s

FAILED (failures=1)


# ====> Load testing
######################################################################


# ====> Clean trash
######################################################################
